### PR TITLE
Enhancement: Use GithubActions instead of Travis for tests

### PR DESCRIPTION
**Pull request by OskarStark**
You can see the first run here: https://github.com/OskarStark/Twig/pull/3

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            - name: "Install PHP with extensions"
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  coverage: "none"
+                  extensions: "gd, pdo_sqlite"
+                  php-version: ${{ matrix.php-version }}
+                  ini-values: memory_limit=-1
+                  tools: composer:v2
+
+            - run: sqlite3 --version
```
Comment (fabpot) [inline .github/workflows/ci.yml]: Is it needed?

Comment (OskarStark) [reply .github/workflows/ci.yml]: No

**Comment by fabpot** [comment]
Can we remove the .travis.yml file?

**Comment by OskarStark** [comment]
> Can we remove the .travis.yml file?

Sure, I forgot, but now done 👍

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**